examples/Polls API.md

```diff
@@ -0,0 +1,179 @@
+FORMAT: 1A
+
+# Polls
+
+Polls is a simple web service that allows consumers to view polls and vote in them.
+
+# Polls API Root [/]
+
+This resource does not have any attributes. Instead it offers the initial API affordances in the form of the links in the JSON body.
+
+It is recommend to follow the “url” link values or Link headers to get to resources instead of constructing your own URLs to keep your client decoupled from implementation details.
+
+## Retrieve the Entry Point [GET]
+
++ Response 200 (application/json)
+
+        {
+            "questions_url": "/questions{?page}"
+        }
+
+## Group Question
+
+Resource related to questions in the API.
+
+## Question [/questions/{question_id}]
+
+A Question object has the following attributes.
+
+- question
+- published_at
+- url
+- choices (an array of Choice objects).
```

Comment:
I would say use the MSON syntax even it it is not in the scope of this blueprint to explain it e.g. 

```
- choices - an array of Choice objects
```

It will be more consistent in the grand picture later